tools/pom.xml

```diff
@@ -34,5 +34,6 @@
     <module>apprunner-common</module>
     <module>apprunner-maven-plugin</module>
     <module>content-generator</module>
+    <module>compatibility</module>
```

Comment:
putting this under `tools` seems rather weird. maybe just put this directly under root?

Reply:
+1